_people/birte_kehr.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Birte Kehr
+category: Contributor
+avatar: /assets/images/people/birte_kehr.jpg
+social_links:
+  email: birte.kehr@decode.is
+  linkedin: birte-kehr-74813175
+---
+
+Dr. Birte Kehr works at the Berlin Institute of Health leading the Junior Research Group Genome Informatics.
```

Comment:
You can also add https://kehrlab.github.io/ if you want to

Reply:
For adding a website I'd have to edit more code, so I just added  github.com/kehrlab/ 
Is that ok ? :)

Reply:
This is also a good idea.
I had 
```suggestion
Dr. Birte Kehr works at the Berlin Institute of Health leading the [Junior Research Group Genome Informatics](https://kehrlab.github.io/).
```
(+ line break) in mind